python/ray/worker.py

```diff
@@ -1382,14 +1384,13 @@ def connect(node,
     worker.gcs_pubsub_enabled = gcs_pubsub_enabled()
     worker.gcs_publisher = None
     if worker.gcs_pubsub_enabled:
-        worker.gcs_publisher = GcsPublisher(
-            channel=worker.gcs_channel.channel())
+        worker.gcs_publisher = GcsPublisher(address=worker.gcs_client.address)
```

Comment:
why not reuse the channel any more?

Reply:
I found out it is unnecessary to explicitly share channel, because GRPC shares the underlying subchannel / connection when channel option is the same. We can verify later in GRPC via channelz.

Reply:
Got it! Btw, could you double confirm this?